packages/query/src/bitcoin/address/address.utils.ts

```diff
@@ -21,3 +22,39 @@ export function createBitcoinCryptoCurrencyAssetTypeWrapper(balance: Money) {
 export function hasInscriptions(utxos: UtxoResponseItem[]) {
   return !isEmptyArray(utxos);
 }
+
+export function calculateOutboundPendingTxsValue(pendingTxs: BitcoinTx[], address: string) {
+  // sum all inputs
+  const sumInputs = sumNumbers(pendingTxs.flatMap(tx => tx.vin.map(input => input.prevout.value)));
+
+  // get all outputs that are sent back to the address
+  const returnedOutputChangeValues = pendingTxs
+    .flatMap(tx => tx.vout.map(output => output))
+    .filter(v => v.scriptpubkey_address === address)
+    .flatMap(output => output.value);
+
+  // sum all filtered outputs
+  const sumOutputs = sumNumbers(returnedOutputChangeValues);
+
+  return sumInputs.minus(sumOutputs).toNumber();
+}
+
+interface UtxoIdentifier {
+  txid: string;
+  vout: number;
+}
+
+export function filterUtxosWithInscriptions(inscriptions: Inscription[]) {
+  return <T extends UtxoIdentifier>(utxo: T) => {
+    return !inscriptions.some(
+      inscription =>
+        `${utxo.txid}:${utxo.vout.toString()}` === `${inscription.txid}:${inscription.output}`
+    );
+  };
+}
+
+export function filterUtxosWithRunes(runes: RunesOutputsByAddress[]) {
+  return <T extends UtxoIdentifier>(utxo: T) => {
+    return !runes.some(rune => rune.output === `${utxo.txid}:${utxo.vout}`);
+  };
+}
```

Comment:
I wonder if these should better live in Bitcoin package? There's no inherent coupling to the API because they're using our Inscription abstraction/model